fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -412,7 +412,26 @@ public void execute() throws Exception {
             } else if (parsedStmt instanceof UseStmt) {
                 handleUseStmt();
             } else if (parsedStmt instanceof CreateTableAsSelectStmt) {
-                handleInsertStmt(uuid);
+                try {
+                    if (execPlanBuildByNewPlanner) {
+                        handleInsertStmtWithNewPlanner(execPlan, ((CreateTableAsSelectStmt) parsedStmt).getInsertStmt());
+                    } else {
+                        handleInsertStmt(uuid);
```

Comment:
If only the new version of the optimizer is supported, then the judgment here is meaningless